eclipse/che
# Remove code that supports namespace strategies other than per-user

### What does this PR do?
Remove code that supports namespace strategies other than per-user
Documentation changes -  in progress....

### Screenshot/screencast of this PR
<img width="1673" alt="Знімок екрана 2021-05-07 о 14 52 38" src="https://user-images.githubusercontent.com/1614429/117445852-e71c6700-af43-11eb-8524-43c4c0ae40d9.png">


### What issues does this PR fix or reference?
Fixes https://github.com/eclipse/che/issues/19536


### How to test this PR?
1. Deploy che-server image `quay.io/skabashn/che-server:che19536`
2. Should create per-user k8s namespace by default
3. Setup `oc patch checluster eclipse-che -n eclipse-che --type=json -p '[{"op": "replace", "path": "/spec/devWorkspace/enable", "value": true}]`
4. Che server startup should fail with message `Only 'per user' is allowed.Using the <username> or <userid> placeholder is required in the 'che.infra.kubernetes.namespace.default' parameter. The current value is: '<workspaceid>-che'`

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
